charts/incubator/lancache-dns/values.yaml

```diff
@@ -1,6 +1,6 @@
 image:
-  repository: tccr.io/truecharts/lancache-dns
-  tag: latest@sha256:0d04cbb893f0b1da8154e2d0a9911b5e9c71f6b2dbb2e66d62e8763b4fe2a17f
+  repository: lancachenet/lancache-dns
+  tag: latest
```

Comment:
Why is that changed?

Reply:
I not sure if this is the latest lancache build thats why i replaced it

Reply:
Our repository mirrors the latest versions, otherwise we hit ratelimits and can't get stats, so we mirror everything and it's generally updated whenever upstream is